tron/core/actionrun.py

```diff
@@ -190,7 +190,7 @@ def __init__(
         self.start_time = start_time
         self.end_time = end_time
         self.exit_status = exit_status
-        self.bare_command = bare_command
+        self.bare_command = maybe_decode(bare_command)
         self.rendered_command = rendered_command
```

Comment:
why do we need it on bare_command but not on rendered_command?

Reply:
iirc it's because we use bare_command as template for `%` operator and that can't be bytes